c10/test/core/DispatchKeySet_test.cpp

```diff
@@ -53,3 +53,53 @@ TEST(DispatchKeySet, Full) {
     ASSERT_TRUE(full.has(tid));
   }
 }
```

Comment:
I'd like to see a test that allocates a dispatch key set with, say, three elements over it, iterates over it, and then shows that what you iterated over is exactly what was in the dispatch key set. All the tests here don't look like they would identify the class of bugs that a test like this would identify.